storage/s3-sdk/noxfile_config.py

```diff
@@ -35,13 +35,13 @@ def get_service_account_email():
 TEST_CONFIG_OVERRIDE = {
     # A dictionary you want to inject into your test. Don't put any
```

Comment:
Sorry added this suggestion in the wrong section before, this should be better.
```suggestion
    "enforce_type_hints": True,
    # A dictionary you want to inject into your test. Don't put any
```